### PR TITLE
First bunch of doc for FCC distributing computing with DIRAC

### PR DESCRIPTION
**Pull request by gganis**

### CODE REVIEW

File: distributed-computing/README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Distributed computing
+
+These pages provide - and dissect - examples of workflows to be run on distributed resources with the [DIRAC Interware system][dirac]
+through [iLCDirac][ilcdirac], the extension developed by the Linear Collider community and used also by CALICE.
+
+The procedure to be enabled to use the FCC resources through DIRAC is first described. Additional information about the use of
+iLCDirac can be found in the [CLIC][wikiclic] and [ILC][wikiilc] dedicated Wiki pages.
+
+Unless specified, in the rest of this section the word `DIRAC` refers to the `iLCDirac` extension introduced above..
```
Comment (vvolkl) [inline distributed-computing/README.md:9]:
```suggestion
Unless specified, in the rest of this section the word `DIRAC` refers to the `iLCDirac` extension introduced above.
```

File: distributed-computing/workflows/Overview.md [hunk 2]
```diff
@@ -0,0 +1,148 @@
+## Overview of the submission scripts
+
+Command line submission to DIRAC is performed using python scripts instantiating the relevant classes.
+The general structure of the script is the following:
+1. Instantiation of the interface to DIRAC;
+1. Creation of a Job manager instance, inclusing input and output sandbox, and all relevant config and data files;
```
Comment (vvolkl) [inline distributed-computing/workflows/Overview.md:6]:
```suggestion
2. Creation of a Job manager instance, including input and output sandbox, and all relevant config and data files; 
```

File: distributed-computing/workflows/Overview.md [hunk 3]
```diff
@@ -0,0 +1,148 @@
+## Overview of the submission scripts
+
+Command line submission to DIRAC is performed using python scripts instantiating the relevant classes.
+The general structure of the script is the following:
+1. Instantiation of the interface to DIRAC;
+1. Creation of a Job manager instance, inclusing input and output sandbox, and all relevant config and data files; 
+1. Creation and configuration of the application to be run and their registration to the job manager instance;
```
Comment (vvolkl) [inline distributed-computing/workflows/Overview.md:7]:
```suggestion
3. Creation and configuration of the application to be run and their registration to the job manager instance;
```

File: distributed-computing/workflows/Overview.md [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+1. Job submission
```
Comment (vvolkl) [inline distributed-computing/workflows/Overview.md:8]:
```suggestion
4. Job submission
```

**Review by vvolkl — APPROVED** [review]
Hi Gerri, I have yet to test it out but the description I find very clear like this, some minor comments inline.

**Comment by gganis** [comment]
Thanks a lot, Valentin. I agree and have approved all of them. Should I merge it? Gerri

**Pull request merged**